R/Plot_Tools.R

```diff
@@ -144,14 +144,52 @@ bc_pal <- function(palette = "primary", reverse = FALSE, ...) {
 scale_color_bc <- function(palette = "primary", discrete = TRUE, reverse = FALSE, ...) {
   pal <- bc_pal(palette = palette, reverse = reverse)
   if (discrete) {
-    # the paste0 arguement is for error messages
+    # the paste0 argument is for error messages
     discrete_scale("colour", paste0("bc_", palette), palette = pal, ...)
   } else {
     scale_color_gradientn(colours = pal(256), ...)
   }
 }
 
-########################################################################################################################*
+# GGPLOT CORROSION FUNCTIONS ----
+#' BC Color Palette with scale_color
+#' @param data Data frame containing a column of named corrosion/scaling indices
+#' @param alpha Set the opacity of the rectangles, default is 0.1
+#' @param fill Set the color of the rectangles, default is "cyan"
+#' @param index_column Name of the columns with corrosion/scaling indices
```

Comment:
Does this work with more than one column? If not, suggested language change below.
```suggestion
#' @param index_column Name of the column in the plot data frame with corrosion/scaling indices names (should be the column used for facetting)
```